app/apphandlers/processorAppHandler.go

```diff
@@ -149,33 +150,38 @@ func (a *processorApp) StartRudderCore(ctx context.Context, options *app.Options
 		jobsdb.WithClearDB(options.ClearDB),
 		jobsdb.WithDSLimit(a.config.gatewayDSLimit),
 		jobsdb.WithSkipMaintenanceErr(config.GetBool("Gateway.jobsDB.skipMaintenanceError", true)),
+		jobsdb.WithStats(stats.Default),
```

Comment:
We can use statsFactory here since we have already defined it. Same for below occurrence as well.
```suggestion
		jobsdb.WithStats(statsFactory),
```